nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/GetHTTP.java

```diff
@@ -62,18 +62,12 @@
 import org.apache.http.impl.client.HttpClientBuilder;
 import org.apache.http.impl.conn.BasicHttpClientConnectionManager;
 import org.apache.http.ssl.SSLContextBuilder;
-import org.apache.nifi.annotation.behavior.InputRequirement;
+import org.apache.nifi.annotation.behavior.*;
```

Comment:
You can set this in **Preferences > Editor > Code Style > Java**, *Imports* tab and then set the following properties:

* Class count to use import with '*': `999`
* Names count to use static import with '*': `999`

You can also import the [Apache NiFi Checkstyle settings](https://cwiki.apache.org/confluence/display/NIFI/Contributor+Guide#ContributorGuide-IntelliJIDEAUsers) into **Other > Checkstyle**.